server/mock/store.go

```diff
@@ -70,6 +71,14 @@ func (ms multiStore) LoadLatestVersion() error {
 	return nil
 }
 
+func (ms multiStore) LoadLatestVersionAndUpgrade(upgrades *store.StoreUpgrades) error {
+	return nil
```

Comment:
should this also panic?
```suggestion
	panic("not implemented")
```

Reply:
It behaves the same as `LoadLatestVersion`. I have no idea why this is implemented or how it should behave, but I seek consistency with the most similar method.

I'm happy if you want to make this all uniform in a different PR.